apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.html.ts

```diff
@@ -154,73 +159,90 @@ export function InstructorAiGenerateDraftEditor({
 function PromptHistory({
   prompts,
   urlPrefix,
+  csrfToken,
 }: {
   prompts: AiGenerationPrompt[];
   urlPrefix: string;
+  csrfToken: string;
 }) {
-  return prompts
-    .filter((prompt) => prompt.prompt_type !== 'auto_revision')
-    .map((prompt) => {
-      return html`<div class="d-flex flex-row-reverse">
-          <div class="p-3 mb-2 bg-secondary text-white rounded">${prompt.user_prompt}</div>
-        </div>
-        <div class="d-flex flex-row">
-          <div class="p-3 mb-2 bg-dark text-white rounded">
-            ${prompt.prompt_type === 'initial'
-              ? 'We generated a potential question.'
-              : 'We have made changes. Please check the preview and prompt for further revisions.'}
-            <div>
-              ${run(() => {
-                if (!prompt.job_sequence_id) return '';
+  return prompts.map((prompt) => {
+    return html`<div class="d-flex flex-row-reverse">
+        <div class="p-3 mb-2 bg-secondary text-white rounded">${prompt.user_prompt}</div>
+      </div>
+      <div class="d-flex flex-row">
+        <div class="p-3 mb-2 bg-dark text-white rounded">
+          ${prompt.prompt_type === 'initial'
+            ? 'We generated a potential question.'
+            : 'We have made changes. Please check the preview and prompt for further revisions.'}
+          <div>
+            ${run(() => {
+              if (!prompt.job_sequence_id || !prompt.id) return '';
 
-                const jobLogsUrl = urlPrefix + '/jobSequence/' + prompt.job_sequence_id;
+              const jobLogsUrl = urlPrefix + '/jobSequence/' + prompt.job_sequence_id;
 
-                return html`
-                  <a class="link-light small" href="${jobLogsUrl}" target="_blank">View job logs</a>
-                `;
-              })}
-            </div>
+              return html`
+                <a class="link-light small" href="${jobLogsUrl}" target="_blank">View job logs</a>
+                <form method="post">
+                  <input type="hidden" name="__action" value="revert_edit_version" />
+                  <input type="hidden" name="unsafe_prompt_id" value="${prompt.id}" />
+                  <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+                  <button class="btn btn-link link-light small d-inline p-0">
```

Comment:
I think the formatting here is ok? response message looks like this: 
<img width="218" alt="image" src="https://github.com/user-attachments/assets/a9a10cfc-6351-46b8-bf1c-95cc915c012e" />